guides/AE2CableManagement.md

```diff
@@ -0,0 +1,71 @@
+
+- Quartz Fibers is the only one out of the six that do not transfer channels, but rather just power. This makes them good for both inscribers and subnetworks, in which subnetworks will be explained later on. They also aren't an individual cable however, as they are an attachment to a cable (as seen in the picture below). They also don't take up a block when connected to other cables.
+- ME Glass Cables is a cable that transmits both power and channels, with the maximum amount of channels allowed being 8 channels. These are very similar to the next cable, and can be interchanged *almost* freely.
+- ME Smart Cables is very similar to Glass cables, however they display the amount of channels shown on the cable, with each line representing one channel. These, alongside the previous cable, are the only ones you can use to connect any AE2 non-full block to the system (Export bus, storage bus, etc...)
+- ME Dense Smart Cable is the same as the previous, however they can hold up to 32 channels on a single line. These cables however can only connect to devices that take up a full block (The full-block variant of ME interfaces, Molecular Assemblers, etc...)
+- Both of the EnderIO conduits serve the same purpose as the ME Smart Cables and ME Dense Cables, however the main differences are that you can not visually see the channels on the cable, and the normal ME Conduit can only connect to devices that take up a full block.
+
+Here is a picture to visually show the different cables/conduits, in order that was described.
+![AE2CablesSideBySide](files/AE2CableManagement/AE2CablesSideBySide.png)
+
+Do note that Glass Cables, Smart Cables, and Dense Smart Cables all have different colored variants. Besides aesethics, different colored cables can not connect to each other, however 'Fluix' Cables can connect to any colored cable, which is how you can connect all the different colored cables together.
+
+## Channels
+
+The first thing, and a very large thing that needs to be understand is how channels work, when they are used, and how to get more, which I will cover in this section.
+
+Channels can only be 'created' by a ME Controller. Each side of the controller can support up to 32 channels, with there being no way to increase the amount of channels per side. You can add multiple controllers to a single network, however they need to all be directly connected to one another, and can not exceed the size of a 7x7x7 block. 
+
+Channels are fairly simple. Any device that serves any purpose within a network, whether it be exporting items, importing items, acting as an interface to the network, whatever it may be, will use a channel. This means if you were to use a Glass Cable to power inscribers, the inscribers will not use channels, as they do not interact with the system besides with power. However, connecting to a 'blank' interface (an interface with nothing in it) will still count as a channel.
```

Comment:
Not exactly true, molecular assemblers do not take a channel and they serve a pretty big purpose in auto-crafting.

Reply:
They actually don't directly interact with the system based on what the wiki says. The specifically have items inserted by the interface and they auto-push to the interface, meaning that it doesn't directly interact with the system, but I should note it nonetheless as it definitely does seem to be that it directly works with the system.